containers/python/rpc.py

```diff
@@ -202,7 +206,7 @@ def predict_strings(self, inputs):
         pass
 
 
-def start(model, ip, port, model_name, model_version, input_type):
+def start(model, host, port, model_name, model_version, input_type):
```

Comment:
As @giulio-zhou mentioned on #37, lets update the args docstring to reflect the change from `ip` to `host`.